test/extended/images/s2i_images.go

```diff
@@ -0,0 +1,122 @@
+package images
+
+import "fmt"
+
+type tc struct {
+	// The image version string (eg. '27' or '34')
+	Version string
+	// The base OS ('rhel7' or 'centos7')
+	Base string
```

Comment:
If that's BaseOS, name the variable accordingly. Base can suggest tons of other things.